libexec/pyenv-which

```diff
@@ -2,27 +2,46 @@
 #
 # Summary: Display the full path to an executable
 #
-# Usage: pyenv which <command> [--nosystem]
+# Usage: pyenv which <command> [--nosystem] [--no-advice]
```

Comment:
`--skip-advice`

Reply:
I ended up going with the `--quiet` flag, but I can change it to `--skip-advice` depending on what maintainers prefer.